canonical-web-and-design/ubuntu.com
# Reduce the blur on the Suru hero on download section

## Done
Reduce the blur on the Suru hero on download section

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download
- Check the left hand suru line is not blurry
